e2e/smoke/BUILD

```diff
@@ -32,3 +33,30 @@ apko_image(
     contents = "@example_lock//:contents",
     tag = "lock:latest",
 )
+
+build_test(
+    name = "test_generated_config",
+    targets = [
+        ":lock_from_generated",
+    ],
+)
+
+apko_config_example(
+    name = "generated_config_with_extra_files",
+    config = ":apko.yaml",
+)
+
+apko_image(
+    name = "image_from_config_with_extra_files",
+    config = ":generated_config_with_extra_files",
+    contents = "@example_lock//:contents",
+    lockfile_basename = "apko.lock.json",
```

Comment:
discussed: It would be good to decouple it from the PR.

Reply:
Done (added separate apko_lock rule instead)